assets/react/v3/shared/hooks/usePortalPopover.tsx

```diff
@@ -173,21 +167,20 @@ interface PortalProps {
 }
 
 export const Portal = ({ isOpen, children, onClickOutside, animationType = AnimationType.slideDown }: PortalProps) => {
-  // We can determine if there any modal opens by using this `hasModalOnStack`
-  // If there any modal opens then we will not set the overflow value to initial.
-  const { hasModalOnStack } = useModal();
-
   useEffect(() => {
     if (isOpen) {
       document.body.style.overflow = 'hidden';
     }
 
     return () => {
-      if (!hasModalOnStack) {
-        document.body.style.overflow = 'initial';
-      }
+      setTimeout(() => {
+        const hasPopoverOnStack = document.querySelectorAll('.tutor-portal-popover').length > 0;
+        if (!hasPopoverOnStack) {
+          document.body.style.overflow = 'initial';
+        }
+      }, 200); // this is the total of -> duration of the animation + threshold + 30ms
```

Comment:
Instead of commenting to recognising a magic value, put the value into a variable.
```ts
const ANIMATION_DURATION_WITH_THRESHHOLD = 200;
```